packages/console/src/console.js

```diff
@@ -92,7 +92,7 @@ export const consoleOmittedProperties = freeze([
  * @typedef {readonly [string, ...any[]]} LogRecord
  *
  * @typedef {Object} LoggingConsoleKit
- * @property {Console} loggingConsole
+ * @property {Partial<Console>} loggingConsole
```

Comment:
Why not the full `Console` type?

Reply:
You're removing methods, so unless you want me to do an unsafe typecast, we cannot use the `Console` type.  It may be better to create a separate type to specify the actual interface exported by loggingConsole, but we'd still need the unsafe typecast because Typescript isn't smart enough to figure out the interface just from looking at your `Object.fromEntries`.

Reply:
I'll try to create a subset-of-console type based on the links you shared.